policy/release/cve.rego

```diff
@@ -141,17 +141,79 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [])
 }
 
+# METADATA
+# title: Deprecated task name
+# description: >-
+#   The `CLAIR_SCAN_RESULT` task name has been deprecated, and has been
+#   replaced with `SCAN_OUTPUT`. If any task results with the old name are
+#   found, this rule will raise a warning.
+# custom:
+#   short_name: deprecated_cve_result_name_warning
```

Comment:
No need to add the `_warning` suffix to the short code.

Reply:
Removed!